engines/python/setup/djl_python/properties_manager/vllm_rb_properties.py

```diff
@@ -51,7 +52,7 @@ class VllmRbProperties(Properties):
     max_lora_rank: Optional[int] = 16
     fully_sharded_loras: bool = False
     lora_extra_vocab_size: int = 256
-    long_lora_scaling_factors: Optional[Tuple[float]] = None
+    long_lora_scaling_factors: Optional[Tuple[float, ...]] = None
```

Comment:
QQ, `...` Is this to allow empty tuple to this?

Reply:
... allow for tuple with variable length. for example, (3.0,) (3.0,4.0)

Tuple[float] only allow for one element in the tuple.